### PR TITLE
Lang By Added

### PR DESCRIPTION
**Pull request by SkiBY**
### Changes proposed in this pull request:

Belarusian Language added

### Status

READY

### How to verify this change

Tests are written, but should be checked by another person

### Additional notes

Small change in RU section for currency BYN

### CODE REVIEW

File: num2words/lang_BY.py [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+}
+
+TENS = {
+    0: 'дзесяць',
+    1: 'адзінаццаць',
+    2: 'дванаццаць',
+    3: 'трынаццаць',
+    4: 'чатырнаццаць',
+    5: 'пятнаццаць',
+    6: 'шастнаццаць',
```
Comment (veryberry) [inline num2words/lang_BY.py:80]: шаснаццаць

File: num2words/lang_BY.py [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+    7: 'семнаццаць',
```
Comment (veryberry) [inline num2words/lang_BY.py:81]: сямнаццаць

File: num2words/lang_BY.py [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+    8: 'васямнаццаць',
+    9: 'дзевятнаццаць',
+}
+
+TWENTIES = {
+    2: 'дваццаць',
+    3: 'трыццаць',
+    4: 'сорак',
+    5: 'пяцьдзясят',
```
Comment (veryberry) [inline num2words/lang_BY.py:90]: пяцьдзесят

File: num2words/lang_BY.py [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+    6: 'шэсцьдзясят',
```
Comment (veryberry) [inline num2words/lang_BY.py:91]: шэсцьдзесят

File: num2words/lang_BY.py [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+        ),
+        'EUR': (('эўра', 'эўра', 'эўра'), ('цэнт', 'цэнты', 'цэнтаў')),
+        'USD': (('долар', 'долары', 'долараў'), ('цэнт', 'цэнты', 'цэнтаў')),
+        'UAH': (('грыўна', 'грыўны', 'грыўнаў'), ('капейка', 'капейкі', 'капеек')),
+        'KZT': (('тэнге', 'тэнге', 'тэнге'), ('тыйін', 'тыйіны', 'тыйінаў')),
+        'BYN': (
+            ('беларускі рубель', 'беларускія рублі', 'беларускіх рублёў'),
+            ('капейка', 'капейкі', 'капеек')
+        ),
+        'UZS': (('сум', 'сума', 'сумаў'), ('тыйін', 'тыйіны', 'тыйінаў')),
```
Comment (veryberry) [inline num2words/lang_BY.py:137]: можа сума -> сумы

File: num2words/lang_BY.py [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+
+    def setup(self):
+        self.negword = 'мінус'
+        self.pointword = 'коска'
+        self.ords = {
+            'нуль': 'нулявы',
+            'адзін': 'першы',
+            'два': 'другі',
+            'тры': 'трэці',
+            'чатыры': 'чацьвёрты',
```
Comment (veryberry) [inline num2words/lang_BY.py:148]: чацвёрты ?

File: num2words/lang_BY.py [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+            'пяць': 'пяты',
+            'шесць': 'шасты',
```
Comment (veryberry) [inline num2words/lang_BY.py:150]: шосты (націск на О)

File: num2words/lang_BY.py (hunk 4 above)
Comment (SkiBY) [reply num2words/lang_BY.py:91]: шэсцьдзясят - новыя правiлы

File: num2words/lang_BY.py (hunk 5 above)
Comment (SkiBY) [reply num2words/lang_BY.py:137]: а хто ведае дакладна?...
але сумы гучыць... лепей )

Comment (veryberry) [reply num2words/lang_BY.py:137]: калі гэта мужчынскі род (бо такі канчатак) то
1 сум 2 сумы

**Comment by SkiBY** [comment]
@veryberry please recheck

**Review by veryberry — APPROVED** [review]

**Comment by veryberry** [comment]
reviewed

**Review by veryberry — DISMISSED** [review]
reviewed

**Comment by SkiBY** [comment]
@mrodriguezg1991 
could you check it please

File: num2words/lang_BY.py [hunk 8]
```diff
@@ -0,0 +1,320 @@
...
+TENS = {
+    0: 'дзесяць',
+    1: 'адзінаццаць',
+    2: 'дванаццаць',
+    3: 'трынаццаць',
+    4: 'чатырнаццаць',
+    5: 'пятнаццаць',
+    6: 'шаснаццаць',
+    7: 'сямнаццаць',
+    8: 'васямнаццаць',
```
Comment (yorlov) [inline num2words/lang_BY.py:82]: [васемнаццаць](https://ru.wiktionary.org/wiki/%D0%B2%D0%B0%D1%81%D0%B5%D0%BC%D0%BD%D0%B0%D1%86%D1%86%D0%B0%D1%86%D1%8C)

File: num2words/lang_BY.py [hunk 9]
```diff
@@ -0,0 +1,320 @@
...
+        self.negword = 'мінус'
+        self.pointword = 'коска'
+        self.ords = {
+            'нуль': 'нулявы',
+            'адзін': 'першы',
+            'два': 'другі',
+            'тры': 'трэці',
+            'чатыры': 'чацвёрты',
+            'пяць': 'пяты',
+            'шесць': 'шосты',
```
Comment (yorlov) [inline num2words/lang_BY.py:150]: шэсць

File: num2words/lang_BY.py [hunk 10]
```diff
@@ -0,0 +1,320 @@
...
+            'сем': 'сёмы',
+            'восем': 'восьмы',
+            'девяць': 'дзявяты',
```
Comment (yorlov) [inline num2words/lang_BY.py:153]: дзевяць

File: num2words/lang_BY.py [hunk 11]
```diff
@@ -0,0 +1,320 @@
...
+        }
+
+        self.ords_adjective = {
+            'адзін': 'адна',
+            'адна': 'адна',
+            'дзве': 'двух',
+            'тры': 'трох',
+            'чатыры': 'чатырох',
+            'пяць': 'пяці',
+            'шесць': 'шасці',
```
Comment (yorlov) [inline num2words/lang_BY.py:165]: шэсць

File: num2words/lang_BY.py [hunk 12]
```diff
@@ -0,0 +1,320 @@
...
+            'сем': 'сямі',
+            'восем': 'васьмі',
+            'дзевяць': 'дзевяті',
```
Comment (yorlov) [inline num2words/lang_BY.py:168]: дзевяцi

**Review by yorlov — CHANGES_REQUESTED** [review]

File: num2words/lang_BY.py (hunk 10 above)
Comment (SkiBY) [reply num2words/lang_BY.py:153]: дзевяць - так, дзявяты па новых правiлах

**Comment by SkiBY** [comment]
@yorlov updated, corrected, some gender bugs corrected thanks to DD
coverage increased, ready to be approved, I hope

**Review by Spaider — APPROVED** [review]
Denis is here, Belarusian-speaking guy.
This MR looks good to me.

**Comment by SkiBY** [comment]
@mrodriguezg1991 could you please recheck it?

**Comment by SkiBY** [comment]
@mrodriguezg1991 could you please approve it?

**Comment by mrodriguezg1991** [comment]
> @mrodriguezg1991 could you please approve it?

@SkiBY you need to enable isort and flake validation in order to pass the test, the test for this MR were successful because the validation was removed, once this is solve i will approve, sorry for any inconvenience and thanks for taking the time to contribute to the project

**Comment by SkiBY** [comment]
> > @mrodriguezg1991 could you please approve it?
> 
> @SkiBY you need to enable isort and flake validation in order to pass the test, the test for this MR were successful because the validation was removed, once this is solve i will approve, sorry for any inconvenience and thanks for taking the time to contribute to the project

reformatted 
flake8 and isort locally passed

**Comment by SkiBY** [comment]
@mrodriguezg1991 could you please review this PR?

**Comment by mrodriguezg1991** [comment]
> @mrodriguezg1991 could you please review this PR?

Hello, Can you please review previous comments? This PR removes the isort and flake8 validation, you need too fix that in order to merge this changes, thanks for taking the time to contribute

**Comment by SkiBY** [comment]
> > @mrodriguezg1991 could you please review this PR?
> 
> Hello, Can you please review previous comments? This PR removes the isort and flake8 validation, you need too fix that in order to merge this changes, thanks for taking the time to contribute

Sorry @mrodriguezg1991 , I read it in opposite way. 
tox.ini restored

**Comment by SkiBY** [comment]
@mrodriguezg1991  touched code for re-run - all checks are passed. Probably, yesterday there were problems with Actions?

**Review by mrodriguezg1991 — APPROVED** [review]

**Pull request merged**